src/azul/plugins/metadata/anvil/service/response.py

```diff
@@ -114,7 +114,8 @@ def choose_entry(_term):
             for bucket in agg['myTerms']['buckets']
         ]
 
-        # Add the untagged_count to the existing termObj for a None value, or add a new one
+        # Add the untagged_count to the existing termObj for a None value, or add a new
+        # one
```

Comment:
```suggestion
        # Add the untagged_count to the existing termObj for a None value, or
        # add a new one
```
This should match how this same line is wrapped in `plugins/metadata/hca/service/response.py`